3p/README.md

```diff
@@ -50,6 +50,8 @@ The better an ad network does on the above requirements, the earlier ads from it
 
 We are also excited to start conversations how modern web tech could improve overall ads latency, memory usage and framerate impact and how AMP's open source model could be used to inject trust through an auditable common code base into the ads ecosystem which would reduce the necessity of redundant metrics collection.
 
+(Further details for ad integrations.)[../ads/README.md]
```

Comment:
I'd suggest saying something like "Review the [ads/README] for further details  on ad integration." for better flow